deploy/clowdapp.yaml

```diff
@@ -109,8 +119,23 @@ parameters:
   - description: Determines Clowder deployment
     name: CLOWDER_ENABLED
     value: "true"
-  - description: Determines if Cloud Watch logging is enabled
-    name: ENABLE_CLOUDWATCH_LOGGING
+  - description: AWS CloudWatch logging integration
+    name: CLOUDWATCH_ENABLED
     value: "false"
+  - description: AWS instance prefix adds string to all instance names, leave blank for production
+    name: AWS_INSTANCE_PREFIX
+    value: ""
```

Comment:
This is where @RHEnVision/qe might be interested - set this to "test-" to have all instances prefixed so they can be cleaned out with Cloudwash.